resources/settings.xml

```diff
@@ -6,7 +6,7 @@
     <setting id="logout" type="action" label="30017" action="RunPlugin(plugin://plugin.video.netflix/action/logout/)" option="close"/>
     <setting id="adultpin_enable" type="action" label="30062" default="False" action="RunPlugin(plugin://plugin.video.netflix/action/toggle_adult_pin/)" option="close"/>
     <setting label="30053" type="lsep"/>
-    <setting id="autologin_enable" type="bool" label="30054" default="false"/>
+    <setting id="autologin_enable" type="bool" label="30054" default="false" enable="eq(0,true)"/>
     <setting id="autologin_user" type="text" label="30055" default="" enable="false" visible="eq(-1,true)" subsetting="true"/>
     <setting id="autologin_id" type="text" label="30056" default="" enable="false" visible="eq(-2,true)" subsetting="true"/>
     <setting id="info" type="text" label="30057" default="" enable="false" visible="eq(-3,true)"/>
```

Comment:
in this case i would put the help id "info", above id "autologin_enable" and always visible

Reply:
I'm not sure I get your point sorry :/
But it's not just an info, as it allosw disabling the autologin.

Reply:
voici
![image](https://user-images.githubusercontent.com/3257156/61483556-57d44700-a99d-11e9-93db-761753e345e5.png)

Reply:
help to understand better how set autologin

Reply:
Changed as requested!